content/en/docs/tasks/administer-cluster/kubelet-config-file.md

```diff
@@ -117,12 +122,75 @@ kind: KubeletConfiguration
 address: "192.168.0.8"
 ```
 
+The resulting configuration will be as follows:
+```yaml
+apiVersion: kubelet.config.k8s.io/v1beta1
+kind: KubeletConfiguration
+port: 20250
+serializeImagePulls: false
+evictionHard:
+  memory.available: "200Mi"
+address: "192.168.0.8"
+```
+Additionally, you can overidde the slices/lists values of the kubelet configuration. For example, you can override the `clusterDNS` and `featureGates` fields as follows:
+
+Main kubelet configuration file contents:
+```yaml
+apiVersion: kubelet.config.k8s.io/v1beta1
+kind: KubeletConfiguration
+port: 20250
+serializeImagePulls: false
+evictionHard:
+  memory.available: "200Mi"
+clusterDNS:
+  - "192.168.0.9"
+  - "192.168.0.8"
+featureGates:
+  MemoryQoS: true
+```
+
+Contents of a file in `--config-dir` directory:
+```yaml
+apiVersion: kubelet.config.k8s.io/v1beta1
+kind: KubeletConfiguration
+clusterDNS:
+  - "192.168.0.2"
+  - "192.168.0.3"
+  - "192.168.0.5"
+featureGates:
+  MemoryQoS: false
+  KubeletTracing: true
+```
+
+The resulting configuration will be as follows:
+```yaml
+apiVersion: kubelet.config.k8s.io/v1beta1
+kind: KubeletConfiguration
+port: 20250
+serializeImagePulls: false
+evictionHard:
+  memory.available: "200Mi"
+clusterDNS:
+  - "192.168.0.2"
+  - "192.168.0.3"
+  - "192.168.0.5"
+featureGates:
+  MemoryQoS: false
```

Comment:
if you didn't specify MemoryQoS in the drop in config - will it stay true or will change to whatever is default in k8s?

Reply:
If you don't mention the `MemoryQoS` field in the drop-in config, it will display the value mentioned in the base kubelet config unless you set the value via a command-line flag, which takes higher precedence over the drop-in config values.